### DISCUSSION TITLE
Create Editor For Custom Rulests

### DISCUSSION THREAD

**Discussion by SealsarecuteR** [ppy/osu#18266] [Ideas] [answered]
This is probably very hard to implement, especially since you don't create these rulesets and you have to manually download them instead of them being in the 4 basic rulesets that automatically download with the game, but I think it would be cool to make charts in an editor for custom rulesets, instead of them being converted, and be able to play other peoples maps, just like the other 4 rulesets.

**Comment by bdach** [comment, marked as answer]
ruleset authors will actually be able to do this themselves. the APIs to allow own composers for custom rulesets are already partially in place.

one place where support is lacking is saving the beatmaps to the legacy format. this will need figuring out and is low priority for the time being.
